gph/python/ripser_interface.py

```diff
@@ -444,16 +470,31 @@ def ripser_parallel(X, maxdim=1, thresh=np.inf, coeff=2, metric="euclidean",
             "`collapse_edges` and `return_generators`cannot both be True."
         )
 
+    is_sparse_matrix_symmetric = None
```

Comment:
```suggestion
    is_sparse_matrix_symmetric = False
```

Is there a reason why initialize with `None` ?

Reply:
It's not a strong reason, but it is to indicate to the reader that this is an optional thing, i.e. it does not apply to the dense cases obviously.

Reply:
Okay, I find it weird, but if this is a convention, then lets keep it with `None`

Reply:
I could maybe name it `is_dm_sparse_and_symmetric` so I don't mean "is the sparse matrix symmetric" but "is the distance matrix sparse and symmetric", and this variable makes sense in all cases.  What do you think?

Reply:
Definitely, the name is better.
And it makes more sense.

BTW, could we move some  `use_sparse_computer` declaration at the beginning after the check ?
So we have the declaration in the same part of the code.

Reply:
I'll change the name, but could you indicate which line you'd like `use_sparse_computer` in?

Reply:
Just below or before `is_dm_sparse_and_symmetric` declaration.
I think it will be better for reader to centralize when it make sense variable declaration.
And also in my opinion to make code more readable.

Reply:
Ok!

Reply:
Both changes done in 34fb487. I let you resolve.